winui/Radial-Gauge/content-pointer.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: Marker Pointer in WinUI Radial Gauge control | Syncfusion
+description: Learn here all about Marker Pointer feature of Syncfusion WinUI Radial Gauge control with image, text pointer support and more.
+platform: WinUI
+control: SfRadialGauge
+documentation: ug
+---
+
+# Shape Pointer in WinUI Radial Gauge
+
+The `ContentPointer` in `SfRadialGauge` allows you to use any content using the `Content` property as a pointer to mark a specified value.
```

Comment:
We have API link for RadialGauge. Please include it.

Reply:
API link for SrRadialGauge, added.